packages/teleport-types/src/generators.ts

```diff
@@ -139,7 +139,9 @@ export interface GeneratorOptions {
   designLanguage?: {
     tokens?: UIDLDesignTokens
   }
-  resources?: { items: UIDLResources['items'] } & { path: ProjectStrategy['resources']['path'] }
+  resources?: { items: UIDLResources['items']; cache: UIDLResources['cache'] } & {
```

Comment:
So....each resource will have it's own cache validator? Do I understand correctly?

Reply:
They need to be defined in the project level for now. This is just to pass to all the `component-plugins` by taking from the `global`. So, each component can use that value to set the timeout in `revalidate`

Reply:
@ionutpasca is this response clear? please resolve again if you agree.